src/views/dialogs/SharePNLAnalyticsDialog.tsx

```diff
@@ -104,7 +104,7 @@ export const SharePNLAnalyticsDialog = ({
       >
         <div tw="flexColumn h-full">
           <div tw="row mb-0.75 gap-0.5">
-            <AssetIcon symbol={assetId} tw="h-[1.625rem]" />
+            <AssetIcon symbol={assetId} tw="h-[1.625rem] w-[1.625rem]" />
```

Comment:
I think we should use `--asset-icon-size` here instead cuz it sets the minWidth and minHeight as well; if we don't wanna do custom css we could add a `size` prop the way `Icon` has it